src/components/Cart/CartProducts/ProductsDetails.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="np-productsDetails">
+    <p class="np-productsDetails__quantity text-secondary-light">{{`${item.quantity} x`}}</p>
+
+    <span v-if="item.name.length < 20"
+          class="np-productsDetails__name">
+
+        {{ item.name.toUpperCase() }}
```

Comment:
I think it would be cleaner to write a function which returns uppercased name with .. concatenated depending on the number of chars